lib/src/bar-chart/types.ts

```diff
@@ -0,0 +1,44 @@
+type DataTypes = string | number | Date;
```

Comment:
There's no use for Date if the user is unable to format it.
![image](https://github.com/dxc-technology/halstack-react/assets/12238686/2c4fd7ba-ce58-4165-9b7f-f80ca83f5d2f)
I think we should remove Date from our valid types and force the user to format it into a string.

Reply:
Another  issue with not being able to format the values is that you are not able to add a prefix or suffix to the Y domain, such as "100K" instead of "100000".

Reply:
I know this, but providing our users with a formatting prop as Cloudscape does can be a bit overwhelming. I don't like either the idea of formatting us by default the Date values. Do you have any suggestions?

Reply:
To format Date values I think the best approach is not to accept Date as a valid type. This will force the user to format it into a string, for example:

![image](https://github.com/dxc-technology/halstack-react/assets/12238686/4e6f97af-e669-407e-b733-77750fcdf504)

Reply:
I was considering the same idea! I'm going to restrict the prop type.